doc/lstm.txt

```diff
@@ -174,7 +174,7 @@ be computed with :
 
 .. math::
 
-    z = \sigma(W x_t + U h_{t-1} + b)
+    z = W x_t + U h_{t-1} + b
```

Comment:
The code for computing z looks like this:

``
        preact = tensor.dot(h_, tparams[_p(prefix, 'U')])
        preact += x_
``
https://github.com/lisa-lab/DeepLearningTutorials/blob/master/code/lstm.py#L175-L176